db/migrate/20200908120905_rebuild_taxonomy_search.rb

```diff
@@ -0,0 +1,7 @@
+class RebuildTaxonomySearch < ActiveRecord::Migration[6.0]
+  def up
+    Filter.where.not(taxonomy_search: nil).find_in_batches do |filters|
+      filters.each(&:save)
```

Comment:
nitpick for future - you could do
```suggestion
    Filter.where.not(taxonomy_search: nil).find_each(&:save)